### DISCUSSION TITLE
Is JWT the only way or good old session cookies are possible?

### DISCUSSION THREAD

**Discussion by Yagger** [nhost/hasura-backend-plus#392] [Q&A] [answered]
JWT is an overhead for most of my projects. Can I configure to use sessions instead?

**Comment by elitan** [comment, marked as answer]
No we only use JWT tokens.

What do you think is the overhead with JWT? Maybe this libraries could help you:

https://github.com/nhost/nhost-js-sdk
https://github.com/nhost-examples

**Reply by Yagger** [reply]
Not big deal. Just JWTs are for distributed systems and give no benefit for mono-server setup, while still require extra work on the front-end or an extra library dependency.

But I just wanted to make sure it is not possible with nhost/hasura-backend-plus right now. Thanks for your reply!

**Reply by elitan** [reply]
Right, our libraries are streamlined to work with HBP and Hasura: [nhost-js-sdk](https://github.com/nhost/nhost-js-sdk), [@nhost/react-apollo](https://github.com/nhost/react-apollo) and [@nhsot/react-auth](https://github.com/nhost/react-auth).

Just to add some nuance to JWTs:

Without JWT you would have to set up an extra webhook for Hasura to authorize requests. Which would add work and latency to each graphql api request.

The JWT approach also works well on platforms where cookies does not work or are unstable. Ex [Chrome is going to phase out third-party cookies by 2022](https://blog.chromium.org/2020/01/building-more-private-web-path-towards.html).
